tests/TCP_echo_client.py

```diff
@@ -231,7 +285,7 @@ def run(self):
 
         self.is_running = False
 
-    def wait(self, timeout=TIMEOUT):
+    def wait(self, timeout=TIMEOUT * 2):
```

Comment:
ctest will fail the test if it takes > TIMEOUT anyway, why is TIMEOUT * 2 used?  Does it actually take 2 * 60 seconds to shut down the client?  If so that seems like a bug to me.